sensu/transport/rabbitmq/rabbitmq.go

```diff
@@ -95,6 +157,25 @@ func (t *RabbitMQTransport) Connect() error {
 		return err
 	}
 
+	if prefetchString := config.Prefetch.String(); prefetchString != "" {
```

Comment:
Instead of the `.String()` method, can we use the `.Int64()` to avoid the explicit string conversion?